src/main/java/com/team766/controllers/pidstate.java

```diff
@@ -0,0 +1,11 @@
+package com.team766.controllers;
+
+abstract class pidstate {
```

Comment:
class name should be `PIDState`

Reply:
class should also probably be Public

Does it necessarily need to be abstract?
Could also be a static class
or regular class with private constructor to prevent instantization

Reply:
If that's allowed, then it sure wouldn't be required. I'm not quite sure if there is other syntax that goes with that though, so let me know if all it is is deleting the class declaration

Reply:
just delete the class and put the enum by itself

Reply:
you'll need to rename the file to match the name of the enum, since it becomes the top-level declaration in the file